README.md

```diff
@@ -24,19 +24,22 @@ When ready, head to the [installation](https://kopia.io/docs/installation/) page
 Pick the Cloud Storage Provider You Want
 ---
 
-Kopia supports saving your [encrypted](https://kopia.io/docs/features/#end-to-end-zero-knowledge-encryption) and [compressed](https://kopia.io/docs/features/#compression) snapshots to all of the following [cloud storage](https://kopia.io/docs/features/#save-snapshots-to-cloud-network-or-local-storage):
+Kopia supports saving your [encrypted](https://kopia.io/docs/features/#end-to-end-zero-knowledge-encryption) and [compressed](https://kopia.io/docs/features/#compression) snapshots to all of the following [storage locations](https://kopia.io/docs/features/#save-snapshots-to-cloud-network-or-local-storage):
 
-* Amazon S3 and any cloud storage that is compatible with S3 (e.g., Wasabi, IDrive E2, DigitalOcean Spaces, and more)
+* Amazon S3 and any cloud storage that is compatible with S3
+  * Including Alibaba Cloud, Amazon Lightsail, Backblaze B2, China Mobile Cloud, Cloudflare R2, Contabo, DigitalOcean Spaces, Dreamhost, Google Cloud Storage, IBM Cloud, IDrive E2, Linode, Mail.ru Cloud, MEGA.io S4, MinIO, Oracle Cloud Infrastructure, OVH, Scaleway, Storj, Synology C2, Tencent Cloud, Vultr, Wasabi, Yandex Cloud, and many more!
```

Comment:
BTW some of them don't work, Storj for example. I would revert this change.

Reply:
Do we know which of these are known not to work? All of these claim to be fully S3 compliant, so all should work with Kopia. I also cross-referenced this list with the Rclone list of S3 compliant storages that work through Rclone's S3 remote. Of the list, either I or someone else have confirmed Kopia's S3 repo to work with: Amazon Lightsail, Backblaze B2, Cloudflare R2, DigitalOcean Spaces, Google Cloud Storage, IDrive E2, Linode, MinIO, Oracle Cloud Infrastructure, OVH, Scaleway, Storj, Vultr, and Wasabi.

Do you happen to know when it was that Storj was not working? Perhaps they did not fully implement S3 compatibility in the past and now have, as Ive used Storj before and just tested it now as well. I am able to successfully create and restore a snapshot and also run quick and full maintenance. They do the whole distributed storage thing but they claim to give instant access to files, so it is not like Amazon Deep Glacier.

If you feel strongly about it, I will remove this list. However, the reason I included the list is (1) SEO and (2) there is an incremental benefit of telling new users that their favorite cloud storage (e.g., Wasabi) works with Kopia rather than just saying all S3-compatible storages work. Perhaps a compromise is only to list the ones that have been confirmed to work. 

LMK.

Reply:
I prefer not to list any particular ones, as that implies we've tested this. 

The compatibility is about more than S3 protocol support, which they all tend to do to a certain extent, but given how different those backends are there's usually more to it so those providers tend to "bend" the protocol a bit to their needs. It's usually about subtle things like read-after-write and list-after-write consistency, server-assigned timestamps, prefix listing support (IIRC this is where Storj failed).

I think this belongs in the community section (i.e. forum). We can start a thread, pin it and folks can post their configuration for various provifders that have been tested.